docs/chapters/flow/index.md

```diff
@@ -529,4 +529,26 @@ Note that more than one `queryset` can be passed with any request.
 ```python
     with Flow().load_config('flow.yml') as search_flow:
         search_flow.search(input_fn=docs, output_fn=print_results, queryset=[top_k_queryset])
-```
\ No newline at end of file
+```
+
+## The Score Field
+
+In ranking document matches, Jina uses several algorithms to compute a `score` field. This field can mean either *distance* (with "smaller is better") or *similarity* (with "higher is better"). As, an example, the `NumpyIndexer` uses k-NN (with various distance metrics) to calculate a *distance*. On the other hand, the `MinRanker` uses a the function *1/(1+s)* (where *s* is the min. score from all `chunks`) in order to "bubble up" the score from child `chunks` to the parent, thus "larger is better".
```

Comment:
It is not only `distance` or `similarity`, it can also mean `relevance`. It is just a minor detail, but needs to be included, because `score` can mean whatever is given

Reply:
I would describe that a lot of rankers can describe the `score` as a measure of `relevance` of a `document` in relation to a given query, and then describe the `MinRanker` (although it will be deprecated soon to be changed by SimpleAggregateRanker`)

Reply:
But isn't **relevance** another type of **similarity** in the sense that they are both positive (i.e. "higher is better") ?

Reply:
yes in the sense of `higher the better`, but it does not relate to "comparision" so directly

Reply:
Updated